privacyidea/models.py

```diff
@@ -3267,3 +3267,12 @@ def save(self):
         else:
             ret = self.id
         return ret
+
+
+class NodeName(db.Model, TimestampMethodsMixin):
+    __tablename__ = "nodename"
+    # TODO: we can use the UUID type here when switching to SQLAlchemy 2.0
+    #  <https://docs.sqlalchemy.org/en/20/core/custom_types.html#backend-agnostic-guid-type>
+    id = db.Column(db.Unicode(36), primary_key=True)
+    name = db.Column(db.Unicode(100), index=True)
+    lastseen = db.Column(db.DateTime(False), index=True, default=datetime.now(tz=tzutc()))
```

Comment:
In other occasions we have

    timestamp = db.Column(db.DateTime, default=datetime.utcnow(), index=True)

Why does this have to be different?

Reply:
See https://blog.miguelgrinberg.com/post/it-s-time-for-a-change-datetime-utcnow-is-now-deprecated

But as i was reading the article further, we should probably stick to using naive UTC timestamps with the type decorator shown here: https://docs.sqlalchemy.org/en/20/core/custom_types.html#store-timezone-aware-timestamps-as-timezone-naive-utc

Reply:
Done in c0c5f2d5e